src/components/Incorporation/ShareStructure.vue

```diff
@@ -54,62 +54,75 @@
                       </v-row>
                     </template>
                   </v-radio>
-                  <v-radio :value="true" label="No maximum" id="lbl-no-maximum" v-if="isNoMaxSharesVisible" />
+                  <v-radio
+                    v-if="isNoMaxSharesVisible"
+                    id="lbl-no-maximum"
+                    :value="true"
+                    label="No maximum"
+                  />
                 </v-radio-group>
 
                 <v-divider class="mt-2 mb-4" />
 
                 <v-radio-group
+                  v-show="isClass"
                   v-model="hasNoParValue"
                   column
                   class="radio-group"
-                  @change="changeParValueFlag()" v-show="isClass"
+                  @change="changeParValueFlag()"
                 >
-                  <v-radio :value="false" id="radio-par-value">
-                    <template v-slot:label>
+                  <v-radio
+                    id="radio-par-value"
+                    :value="false"
+                  >
+                    <template #label>
                       <v-row>
                         <v-col cols="6">
                           <v-text-field
-                            filled
-                            label="Par Value"
                             id="class-par-value"
                             v-model.number="shareStructure.parValue"
+                            filled
+                            label="Par Value"
                             :rules="getParValueRule()"
                             hint="Enter the initial value of each share"
                             persistent-hint
                           />
                         </v-col>
                         <v-col cols="6">
                           <v-select
+                            id="class-currency"
+                            v-model="shareStructure.currency"
                             :items="getCurrencyList()"
                             filled
                             label="Currency"
-                            v-model="shareStructure.currency"
                             :rules="getCurrencyRule()"
                             item-text="`${data.item.name}, ${data.item.code}`"
                             item-value="code"
-                            id="class-currency"
                           >
-                            <template slot="selection" slot-scope="data">
-                               {{ data.item.name }} ({{ data.item.code }})
+                            <template #selection="data">
+                              {{ data.item.name }} ({{ data.item.code }})
                             </template>
-                            <template slot="item" slot-scope="data">
+                            <template #item="data">
```

Comment:
This and line 102 are new replacements that I don't I've seen before. Please test both.

Reply:
Just tested them. They work. And this usage can be found in section `Named Scoped Slots` at https://vuejs.org/guide/components/slots.html#named-slots

![image](https://github.com/bcgov/business-create-ui/assets/60866283/db6252ae-bf91-4334-b31c-868b406d2396)

Reply:
Thanks for testing and finding that reference. We get smarter every day :)